deploy/.env.example

```diff
@@ -39,6 +39,16 @@ REQUIRED_NUMBER_OF_VALIDATORS=1
 #If several validators are used, list them separated by space without quotes
 #E.g. VALIDATORS=0x 0x 0x
 VALIDATORS=0x
+#for erc_to_native mode
+#Set to true if RewardableValidators will be used and fee will be charged
+REWARDABLE_VALIDATORS=false
```

Comment:
Let's have two variables: `HOME_REWARDABLE` and `FOREIGN_REWARDABLE` so we can manage both parts of the bridge. In this particular revision if `FOREIGN_REWARDABLE` is set to `true`, the deployment process should stop and report that this mode is not supportable.

Reply:
Done https://github.com/poanetwork/poa-bridge-contracts/pull/134/commits/228aaf183ab278c18fefc7a6e44b3c934f06425c